src/oscar/templatetags/reviews_tags.py

```diff
@@ -36,3 +38,7 @@ def may_vote(review, user):
 @register.filter
 def is_review_permitted(product, user):
     return product and product.is_review_permitted(user)
+
+@register.filter
+def get_reviews(product):
```

Comment:
A better name would be `approved_reviews`. You could perhaps pass in the number of reviews wanted too instead of piping to the `slice` filter.